io/driver/module_unload_load.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+CONFIG_FILE="$AVOCADO_TEST_DATADIR"/config
+DRIVER=`lspci -k | grep -iw "Kernel driver in use" | cut -d ':' -f2 | sort | uniq`
+
+#Loads the kernel driver modules
+module_load() {
+    echo "Reloading driver $1"
+    modprobe $1
+    if [ $? != 0 ]; then
+        echo "Failed to load driver module $1"
+        break;
+    fi
+    echo
+}
+
+#Unloads the kernel driver modules
+module_unload() {
+    for i in $(cat $CONFIG_FILE | grep "$1=" | awk -F'=' '{print $2}'); do
+        module_unload $i
+        if [ $? != 0 ]; then
+            echo "Failed to unload driver module $i"
+            return
+        fi
+    done
+    echo "Unloaded driver $1"
+    rmmod $1
```

Comment:
There is no check for return number so I'm wondering whether this should be called with `-e`, or if the purpose of this test is not to remove the modules, but only to try to do it and see if the system crashes or not, right? If the later, then avocado has no support for reboot (kernel panics) during execution so the user would only get the partial test results stored in the results dir, without the `json`, `xunit`/... results (basically no results for non-streamlined output plugins).

Would you please clarify what is the purpose of this test?